network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -80,3 +84,31 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
         FieldPanel('contents_title'),
         FieldPanel('notes')
     ]
+
+    @property
+    def is_chapter_page(self) -> bool:
+        """Is this a chapter page (child-Publicationpage). Returns a bool."""
+        parent = self.get_parent().specific
+        return parent.__class__.__name__ == self.__class__.__name__
+
+    def get_chapter_number_or_none(self) -> Union[int, None]:
+        """Return the chapter page number or None."""
+        if self.is_chapter_page:
```

Comment:
I think we need numbers either way, no (not just for chapter page)?

and I'm probably being dense, but if we just render these as an ordered list in html, wont they be numbered anyways? 

depending on the answers to those questions, the answer should probably be the docstring for the function :-D

Reply:
On Chapter pages you can have:
3.1
3.2
3.3

It's the number 3 in this case. 

For Publication Pages we can loop through each loop in the template and use `{{ forloop.counter }}` for numbering, or the numbering that comes with `<ol>`